reports/tasks/msisdn_message_report.py

```diff
@@ -157,8 +157,11 @@ def retrieve_messages(self, ms_client, data, start_date, end_date):
             for message in results:
                 date_sent = datetime.strptime(
                         message['created_at'], "%Y-%m-%dT%H:%M:%S.%fZ")
+                content = message['content']
+                if message['content'] is None:
+                    content = message['metadata'].get('voice_speech_url', None)
```

Comment:
`voice_speech_url` can also be a list. Will that break anything or will it be fine?

Reply:
yeah that'll probably break things
*mutter mutter* stupid json *mutter*